.github/workflows/cd.yaml

```diff
@@ -0,0 +1,26 @@
+name: Tupaia Deploy
+on:
+  workflow_run:
+    # Note: This will only trigger if the workflow file is on the default branch
+    workflows: [Tupaia CI]
+    types: [completed]
+
+permissions:
+  contents: read
+  id-token: write # OIDC token for AWS
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    # Only run if Tupaia CI passed
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: ap-southeast-2
```

Comment:
Went with this way of talking to AWS where no secrets are used as this is how they do it in Tamanu.